akvo/rsr/spa/app/modules/editor/custom-fields.jsx

```diff
@@ -38,7 +38,7 @@ const CustomField = connect(({ editorRdr: { showRequired } }) => ({ showRequired
   }
   return (
     <Item validateStatus={(showRequired && field.mandatory && (value === '' || !value)) ? 'error' : ''} label={<InputLabel optional={!field.mandatory} tooltip={field.helpText}>{field.name}</InputLabel>}>
-      {field.type === 'text' && <Input.TextArea autosize value={value} onChange={({target}) => updateValue(target.value)} />}
+      {field.type === 'text' && <Input.TextArea autosize value={value} onChange={({target}) => updateValue(target.value)} disabled={field.name === 'Optimy Project ID'} />}
```

Comment:
Should we also check the program ID just to be sure that we don't disable an unrelated "Optimy Project ID" field? It is unlikely that another unrelated program has this custom field, though.

Reply:
Let's delay the added complexity when it's really needed.